rasa/cli/x.py

```diff
@@ -243,3 +243,13 @@ def rasa_x(args: argparse.Namespace):
             local.main(args, project_path, args.data, token=rasa_x_token)
```

Comment:
Can we `except AttributeError` here and throw a nicer error/exit with 1 if they run into the deprecation issue? If I train with no `- text` I get the warning upon `rasa x`, which is helpful, but still hidden behind the ugly error:
```
(rasa-x) Ellas-MacBook-Pro-2:moodbot erohmensing$ rasa x
2019-05-29 09:36:06 WARNING  rasa.core.domain  - Deprecated: Templates should not be strings anymore. Utterance template 'utter_happy' should contain either '- text: ' or '- custom: ' attribute to be a proper template.

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

Starting Rasa X in local mode... 🚀
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/rasa", line 11, in <module>
    load_entry_point('rasa', 'console_scripts', 'rasa')()
  File "/Users/erohmensing/rasa/rasa/__main__.py", line 70, in main
    cmdline_arguments.func(cmdline_arguments)
  File "/Users/erohmensing/rasa/rasa/cli/x.py", line 243, in rasa_x
    local.main(args, project_path, args.data, token=rasa_x_token)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/local.py", line 194, in main
    session)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/local.py", line 144, in _initialize_with_local_data
    LOCAL_PROJECT_NAME,
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/api/initialise.py", line 170, in inject_domain
    username=username)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/services/projects_service.py", line 48, in store_domain
    templates = _templates(domain, username, project_id)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/services/projects_service.py", line 259, in _templates
    for template, responses in templates.items() for response in responses
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/rasax/community/services/projects_service.py", line 259, in <listcomp>
    for template, responses in templates.items() for response in responses
AttributeError: 'str' object has no attribute 'get'
```

Reply:
What about 
```
        try:
            local.main(args, project_path, args.data, token=rasa_x_token)
        except Exception as e:
            logger.debug(traceback.format_exc())
            print_error(
                "Sorry, something went wrong. Make sure to start Rasa X with "
                "valid data and valid domain and config files. To see the actual "
                "error start Rasa X with '--debug'. If you need help fixing the issue, "
                "please visit our forum: https://forum.rasa.com/."
            )
        finally:
            process.terminate()
```

Reply:
hm `--debug` doesn't actually show the error though 😄

Reply:
yeah, just tried it... maybe just print the error then in addition to the 
```
            print_error(
                "Sorry, something went wrong. Make sure to start Rasa X with "
                "valid data and valid domain and config files. If you need help fixing the issue, "
                "please visit our forum: https://forum.rasa.com/."
            )
```

Reply:
I think that's a good idea, catching all exceptions as well. Maybe we should add a note in there about checking any other warnings that popped up so that people know to try fixing their templates before they come to the forum since their domain is "valid" (i.e. trains in core)